tools/parity_scale_decoder/src/main.rs

```diff
@@ -0,0 +1,330 @@
+//! Parity Scale decoder tool for Iroha data types. For usage run with `--help`
+
```

Comment:
you can probably make this crate `no_std`

Reply:
I've tried it and that isn't so easily. Some crates I use does not provide `no_std` compatibility, there is no `io` and `fs` analagous in `alloc` or `core` modules and etc. Also I can't see if we really need it. It's probably just a debugging tool

Reply:
nvm then